### PR TITLE
Add and update webdriver tests

### PR DESCRIPTION
**Pull request by chejennifer**
- update timeline webdriver test to check for presence of data
- add webdriver tests for scatter and map tools

### CODE REVIEW

File: server/webdriver_tests/map_test.py [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+from selenium.webdriver.support.ui import Select
+from selenium.webdriver.support import expected_conditions as EC
+import time
+
+MAP_URL = '/tools/map'
+URL_HASH_1 = '#&sv=Median_Age_Person&pc=0&pd=geoId/06&pn=California&pt=State&ept=County'
+PLACE_SEARCH_CA = 'California'
+
+
+# Class to test timeline tool.
```
Comment (shifucun) [inline server/webdriver_tests/map_test.py:26]: map tool

File: server/webdriver_tests/map_test.py [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+class TestScatter(WebdriverBaseTest):
```
Comment (shifucun) [inline server/webdriver_tests/map_test.py:27]: TestMap

**Review by shifucun — APPROVED** [review]
Very nice!!

File: server/webdriver_tests/map_test.py (hunk 1 above)
Comment (chejennifer) [reply server/webdriver_tests/map_test.py]: done

File: server/webdriver_tests/map_test.py (hunk 2 above)
Comment (chejennifer) [reply server/webdriver_tests/map_test.py]: done

**Pull request merged**